setup_cluster.sh

```diff
@@ -76,6 +76,7 @@ function deploy_operators() {
   if [[ ${CLUSTER_TYPE} == "host" ]]; then
     oc new-project "$HOST_OPERATOR_NS"
     NAME=host-operator OPERATOR_NAME=toolchain-host-operator NAMESPACE=$HOST_OPERATOR_NS envsubst < ./config/operator_deploy.yaml | oc apply -f -
+    DOMAIN=$DOMAIN DOMAIN_API_KEY=$DOMAIN_API_KEY NAMESPACE=$HOST_OPERATOR_NS envsubst < ./config/host_operator_secret.yaml | oc apply -f -
```

Comment:
I wonder if `MAILGUN_DOMAIN` and `MAILGUN_API_KEY` would be better names than just `DOMAIN`...

Reply:
oh! sure! i think i saw in the docs that they were prefixing their env vars with MG - so maybe we could do `MG-DOMAIN` and `MG_DOMAIN_API_KEY`?

Reply:
changed here: 9c38295ffcf834aec9d322cbd1b942a97aa4e5f7 MAILGUN is probably more clear so went with your suggestions :-)